### DISCUSSION TITLE
image help

### DISCUSSION THREAD

**Discussion by throne88** [ImageMagick/ImageMagick#5545] [General]
so I have found this image that I know is edited for an ARG only problem is the most important part of it is blacked out could somebody unblack it out for me? 
the image link: ![happyfluffbunny](https://user-images.githubusercontent.com/113188986/189363622-637df724-b52e-404b-8d42-2d7748ff172d.jpg)

**Comment by throne88** [comment]
I want just the telephone number unblocked considering this isn't an actual thing that happened and the phone number probably is just something that will help people doing the ARG it's all from an ARG called "Emet's Crossing" which isn't an actual place

**Reply by throne88** [reply]
also no real police department puts up actual phone numbers and also I am being driven insane by the fact that nothing in the arg makes sense to me

**Comment by fmw42** [comment]
I do not understand what you are asking!  Also what is ARG?

**Comment by snibgo** [comment]
> ...the most important part of it is blacked out could somebody unblack it out for me?

I think you want to know the telephone number that has been blacked out.

That information is lost. The image has a black rectangle, and the file does not contain the pixels as they were before the black rectangle was created.

Perhaps an internet search of similar images may find variations of the same image.
